site/app/controllers/admin/ReportController.php

```diff
@@ -21,7 +22,7 @@
 /**
  * Class ReportController
  * @package app\controllers\admin
- *
+ * @AccessControl(role="INSTRUCTOR")
```

Comment:
What happens if we put "instructor" here (lowercase string)? Does this cause an exception to be thrown?

Reply:
Yes it does.

https://github.com/Submitty/Submitty/blob/43751a196346fb950101ddbeeed214be89ea0bb2/site/app/libraries/routers/AccessControl.php#L43-L58

Reply:
added `strtoupper` to make lowercase also valid